app/modules/integrations/integrations_service.py

```
@@ -1749,8 +1749,8 @@ async def save_integration(
             )
             setattr(db_integration, "unique_identifier", unique_identifier)
             setattr(db_integration, "created_by", user_id)
-            setattr(db_integration, "created_at", datetime.utcnow())
-            setattr(db_integration, "updated_at", datetime.utcnow())
+            setattr(db_integration, "created_at", datetime.now(timezone.utc))
```

Ruff (0.14.14)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)